addons/mail/static/src/views/activity/activity_model.js

```diff
@@ -0,0 +1,17 @@
+/** @odoo-module */
+
+import { RelationalModel } from "@web/views/relational_model";
+
+export class ActivityModel extends RelationalModel {
+    async load(params = {}) {
+        await this.fetchActivityData();
```

Comment:
const activityData = this.fetchActivity()
await super.load();
this.activityData = activityData;